src/dnnl/dnnl_graph_compiler.cpp

```diff
@@ -43,54 +41,112 @@
 
 // dnnl_graph_compiler.h interface implementation.
 
+using namespace mlir;
+
+extern "C" {
+extern int gc_runtime_keep_alive;
+}
+
 struct dnnl_graph_compiler_executable {
-  // TODO: Implement
+  dnnl_graph_compiler_executable(MLIRContext &context,
+                                 const std::string_view &json)
+      : inputIds(), outputIds(), jmod() {
+    auto mod = JsonParser::parse(context, json, inputIds, outputIds, strides);
+    auto jmod = gc::JitModule::create(mod);
```

Comment:
`jmod` is the name of private class member. It would be better to add prefix `_` to class member just as `JsonParser` to avoid confusion.

Reply:
I think, we need a code style convention. Currently, different styles are used in different parts of the project. Regarding the `_` prefix, there are opinions that it's reserved in C++, but I haven't found any evidence of this.